### PR TITLE
Added Nextflow programming lang extension.

### PR DESCRIPTION
**Pull request by pditommaso**
See https://github.com/search?p=1&q=extension%3Anf+process&ref=searchresults&type=Code&utf8=%E2%9C%93

### CODE REVIEW

File: lib/linguist/languages.yml [hunk 1]
```diff
@@ -2325,8 +2325,8 @@ NetLogo:
   extensions:
   - .nlogo
   tm_scope: source.lisp
-  ace_mode: lisp
-
+  ace_mode: lisp 
+
```
Comment (pchaigno) [inline lib/linguist/languages.yml]: These changes look like typos ;)
May be best fixed with `git rebase -i`.

**Comment by arfon** [comment]
Hi @pditommaso, thanks for opening this Pull Request. It looks like there are only ~180 `Nextflow` files on GitHub right now and this doesn't meet our [contributing guidelines](https://github.com/github/linguist/blob/master/CONTRIBUTING.md#adding-a-language) for adding a new language.

Is there a more expansive/inclusive search that could be constructed here to find additional files?

**Comment by pditommaso** [comment]
I don't think so.

p

**Comment by arfon** [comment]
OK, then I think we should close this for now and revisit periodically to assess usage.

**Comment by pditommaso** [comment]
Hello, adoption of nextflow growth considerable in the last year. There are more than 1k hits on Github, see for example:
https://github.com/search?q=extension%3Anf+input&type=Code
https://github.com/search?q=extension%3Anf+output&type=Code

May you revise this PR ?

**Pull request closed without merging**